keyvault/spring-cloud-azure-starter-keyvault-secrets/property-source/src/main/resources/application.yml

```diff
@@ -5,9 +5,9 @@ spring:
         secret:
           property-source-enabled: true
           property-sources:
-            -
-              name: key-vault-1
+            - name: key-vault-property-souece-1
               endpoint: ${ENDPOINT_1}
-            -
-              name: key-vault-2
+            - name: key-vault-property-souece-2
               endpoint: ${ENDPOINT_2}
+  profiles:
+    active: develop
```

Comment:
<img width="341" alt="image" src="https://user-images.githubusercontent.com/4465723/158723888-7bcc0a39-c511-4ab5-8acd-f234e1244933.png">
what's this used for here

Reply:
Used to enable `application-develop.yml`, which is used for local development purpose.